spec/forms/upload_form_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UploadForm, type: :model do
+  describe '#valid?' do
+    context 'when all attributes are valid' do
```

Comment:
Prefer the formulation as the Web chapter uses it more:

```suggestion
    context 'given valid all attributes' do
```